test/support/factory.ex

```diff
@@ -433,10 +434,11 @@ defmodule Trento.Factory do
   end
 
   def sap_system_with_cluster_and_hosts do
-    %ClusterReadModel{id: cluster_id} = insert(:cluster, type: :hana_scale_up, health: :passing)
+    %ClusterReadModel{id: cluster_id} =
+      insert(:cluster, type: ClusterType.hana_scale_up(), health: :passing)
```

Comment:
`Health.passing()` forgotten here. You could append in the health enum PR (there are others in this file)